lib/rspotify/user.rb

```diff
@@ -286,6 +286,78 @@ def saved_tracks?(tracks)
       User.oauth_get(@id, url)
     end
 
+    # Remove albums from the user’s “Your Music” library.
+    #
+    # @param albums [Array<Album>] The albums to remove. Maximum: 50.
+    # @return [Array<Album>] The albums removed.
+    #
+    # @example
+    #           albums = user.saved_albums
+    #
+    #           user.saved_albums.size #=> 20
+    #           user.remove_albums!(albums)
+    #           user.saved_albums.size #=> 0
+    def remove_albums!(albums)
+      albums_ids = albums.map(&:id)
+      url = "me/albums?ids=#{albums_ids.join ','}"
+      User.oauth_delete(@id, url)
+      albums
+    end
+
+    # Save albums to the user’s “Your Music” library.
+    #
+    # @param albums [Array<Album>] The albums to save. Maximum: 100.
+    # @return [Array<Album>] The albums saved.
+    #
+    # @example
+    #           albums = RSpotify::Album.search('Know')
+    #
+    #           user.saved_albums.size #=> 0
+    #           user.save_albums!(albums)
+    #           user.saved_albums.size #=> 20
+    def save_albums!(albums)
+      albums_ids = albums.map(&:id)
+      url = "me/albums"
+      request_body = albums_ids.inspect
+      User.oauth_put(@id, url, request_body)
+      albums
+    end
+
+    # Returns the albums saved in the Spotify user’s “Your Music” library. ** Includes albums whose tracks you saved
+    #
+    # @param limit  [Integer] Maximum number of albums to return. Maximum: 50. Minimum: 1. Default: 20.
+    # @param offset [Integer] The index of the first album to return. Use with limit to get the next set of albums. Default: 0.
+    # @return [Array<Album>]
+    #
+    # @example
+    #           albums = user.saved_albums
+    #           albums.size       #=> 20
+    #           albums.first.name #=> "Do I Wanna Know?"
```

Comment:
`"Do I Wanna Know?"` was for the track so you can replace this for any album name you like :)